ci/jenkins/playbook/roles/packages/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+---
+- name: terraform install
```

Comment:
Same here.. Also, these never got installed on the machine.. Worth checking why.. I had to install it from test_aws.sh script